### PR TITLE
[Issue #494] New config option skipLink for accessibility

### PR DESCRIPTION
**Pull request by dmed256**
Please makes sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you are merging your commits to `master` branch.
* [x] Add some descriptions and refer relative issues for you PR.
* [x] DO NOT include files inside `lib` directory.

---

- Updated prismjs from 1.9.0 to 1.13.0
- #494 Added `skipLink` config option + updated docs

### CODE REVIEW

**Comment by dmed256** [comment]
Seems `href="#?id=main"` only works for `/`, not sure how to generalize it

**Comment by dmed256** [comment]
We might need to add an `onclick` event to properly focus on `#main`

File: src/core/render/index.js [hunk 1]
```diff
@@ -251,6 +258,15 @@ export function initRender(vm) {
     dom.before(navAppendToTarget, navEl)
   }
 
+  if (config.skipLink
```
Comment (dmed256) [inline src/core/render/index.js]: I tried adding this to `<body></body>` but `<nav>` was always added first

**Comment by QingWei-Li** [comment]
Yes.

**Comment by trusktr** [comment]
I think accessibility should be in core, right? Isn't what the site "looks" like important, not just for people with eyes? @jhildenbiddle wdyt?

**Comment by jhildenbiddle** [comment]
@trusktr --

Accessibility should absolutely be a concern for the core team, and we can and should do better moving forward. See https://github.com/docsifyjs/docsify/issues/494#issuecomment-647092391 for related feedback.

I'm guessing that the above recommendation was made based on the assumption that adding a skip link meant that the link would be visible on all sites and, since many people may not want that, users could "opt-in" by either installing the plugin or not. Again, just a guess. Fortunately, this isn't the case. We can add a "Skip Navigation" link to the top of the sidebar that is hidden by default but becomes visible when it receives focus.

Let's work through the problem in #494 and see where we land. I think this is a good feature to add and accessibility features like this should be part of docsify core. That said, I'm not sure this is the sort of thing we should add to v4. A significantly improved a11y experience would also be a pretty big achievement for v5.

**Pull request closed without merging**